OptimizelySDKCore/OptimizelySDKCoreTests/OPTLYNetworkServiceTest.m

```diff
@@ -64,7 +64,7 @@ - (void)testDownloadProjectConfigRequestRetrievesProperDatafileVersion {
     __weak XCTestExpectation *expectation = [self expectationWithDescription:@"testDownloadProjectConfigRequestRetrievesProperDatafileVersion"];
     [self stub200Response];
     
-    NSString *filePath = [NSString stringWithFormat:OPTLY_DATAFILE_URL, kProjectId];
+    NSString *filePath = [OPTLYDatafileConfig defaultProjectIdPath:kProjectId];
```

Comment:
nit: This var name is so confusing. I thought it meant filesystem and I panicked for a bit

Reply:
i will change it but as you can see from my change.  that is the original name.